agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/localstorage/LocalFileLoader.java

```diff
@@ -120,14 +125,19 @@ PersistedFile loadTelemetriesFromDisk() {
       if (!isInstrumentationKeyValid(instrumentationKey)) {
         fileInputStream.close(); // need to close FileInputStream before delete
         if (!FileUtil.deleteFileWithRetries(tempFile)) {
-          operationLogger.recordFailure("Unable to delete file: " + tempFile.getAbsolutePath());
+          operationLogger.recordFailure(
+              "Unable to delete file: " + tempFile.getAbsolutePath(),
+              AzureMonitorMsgId.DISK_PERSISTENCE_READ_ERROR);
```

Comment:
maybe DISK_PERSISTENCE_READ_ERROR -> DISK_PERSISTENCE_LOADER_ERROR, since read error makes it sounds like error reading from disk